dojson/contrib/marc21/utils.py

```diff
@@ -126,7 +126,22 @@ def split_blob(blob):
     raise StopIteration()
 
 
+def split_stream(stream):
+    """Split the stream using <record.*?>.*?</record> as pattern."""
+    buf = []
```

Comment:
> We should make create_record to accept the element of lxml tree element anyway so we don't create and parse strings around.

In INSPIRE we were planning to ship this part down to Celery, so to be able to use the string is still preferable for us since they wouldn't require serialization. Maybe we can support both?

> It's not important at all if you pass the instance down to `create_record`.

Very good point :smile: 

> ￼ method="html" ￼

Damn, it's the only one I haven't checked given html method usually means unpredictable behaviors.